docs/kyma/docs/04-05-use-your-own-image.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Use your own Kyma Installer image
+type: Installation
+---
+
+When you install Kyma from the release, you use the release artifacts that already contain the Kyma Installer, a Docker image containing the combined binary of the Installer and the component charts from the `/resources` folder.
```

Comment:
Maybe this sentence should be split since now it is not clear (to me at least) if the Kyma installer is a Docker image containing [...]  or if it's Kyma installer, a Docker image and so on 🤔  but it's just a thought :)

Reply:
`When you install Kyma from the release, you use the release artifacts that already contain the Kyma Installer - a Docker image containing...` ?